soc/xtensa/intel_adsp/Kconfig

```diff
@@ -12,6 +12,22 @@ config SOC_FAMILY
 	string
 	default "intel_adsp"
 
+config ADSP_LOG_WIN_BASE
+	hex "Address of host logging window"
+	default 0x9e008000
```

Comment:
Fwiw, It's probably not worth doing this for trace when the DMA transport is used (since we manually writeback prior to DMA copy) when using the DMA trace.

Reply:
Yeah, I need to sit down at some point and figure out what exactly the specified protocol is over these regions and who all the users are.  The reason this exists is because I need very early synchronous output[1], which is difficult to manage with the (somewhat heavyweight) logging system.   So I just cloned what the existing code does.  But it should all be unified.  FWIW: I'm happy to leave this in my own tree for now, most users aren't going to care.  I'll finish the cleanup and push a linkage-only implementation that should be good for CAVS 1.5 at least, without too much pollution.

[1] As part of the linker rework, I'm half way to to a mental model for a design where the bootloader and IMR space moves into the main link, which would allow us to get log output all the way back to firmware ROM handoff.